generated/gitops-template/azure/azure-pipelines.yml

```diff
@@ -3,13 +3,24 @@
 trigger:
 - main
 
+# Using self-hosted 'resourcehub' agent pool by default
+# Change accordingly when using a different agent pool
+# Can be deleted if self-hosted agents are not being used
+# See https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/agents
 pool:
   name: resourcehub
```

Comment:
I think this should be `Default` which is the default agent pool Azure provides. `resourcehub` is something I invented to refer to the agent pool I created using our internal resourcehub service.